service/src/test/java/org/sunbird/learner/actors/UserManagementActorTest.java

```diff
@@ -50,11 +51,12 @@
   UserUtil.class,
   InterServiceCommunicationFactory.class,
 })
-@PowerMockIgnore({"javax.management.*"})
+@PowerMockIgnore({"javax.management.*", "javax.crypto.*", "javax.net.ssl.*", "javax.security.*"})
 public class UserManagementActorTest {
 
-  private ActorSystem system = ActorSystem.create("system");
-  private static final Props props = Props.create(UserManagementActor.class);
+  public ActorSystem system = ActorSystem.create("system");
+  public static final Props props = Props.create(UserManagementActor.class);
+  public static UserService userService;
```

Comment:
nothing should be declared public static unless it is final constant variable

Reply:
incorrect - we should not mock user-service it is internal class - using cassandra internally